js-miniapp-sdk/src/event-types/index.ts

```diff
@@ -5,4 +5,6 @@ export enum MiniAppEvents {
   EXTERNAL_WEBVIEW_CLOSE = 'miniappwebviewclosed',
   PAUSE = 'miniapppause',
   RESUME = 'miniappresume',
+  KEYBOARDSHOWN = 'keyboardshown',
+  KEYBOARDHIDDEN = 'keyboardhidden',
```

Comment:
I think these should be pre-fixed with `miniapp` to match the other events and prevent conflicting event names in the mini app:
```suggestion
  KEYBOARDSHOWN = 'miniappkeyboardshown',
  KEYBOARDHIDDEN = 'miniappkeyboardhidden',
```

Reply:
fixed